lib/concordance-options.js

```diff
@@ -124,6 +125,6 @@ const plainTheme = cloneDeepWith(colorTheme, value => {
 });
 
 const theme = chalk.enabled ? colorTheme : plainTheme;
-exports.default = {maxDepth: 3, plugins, theme};
+exports.default = {maxDepth: util.inspect.defaultOptions.depth, plugins, theme};
```

Comment:
What is default value for this variable? I think you should update also documentation.

Reply:
The existing documentation never mentions what is the current depth (`git grep -i depth docs` returns empty), so I am not sure where I should update this information. Any suggestions where I could include it?

Reply:
The default is `2`, but it was `20` for some of the Node.js 11 releases.

I don't think we should go lower than 3, but we could use `Math.min(3, util.inspect.defaultOptions.depth)`. More recent Node.js 11 releases seem to be using `2` again so that's OK — we don't need to support early versions of Node.js 11.

Reply:
I believe all comments in this thread have been addressed (added docs, enforced min depth of 3, added getter).